gui/packages/ubuntupro/lib/l10n/app_en.arb

```diff
@@ -51,5 +51,30 @@
     "purchaseStatusUserGaveUp":"Purchase attempt was canceled.",
     "purchaseStatusNetwork":"A network issue prevented the purchase to proceed. Please try again once connected.",
     "purchaseStatusServer":"Something went wrong with Microsoft Store. Please try again later.",
-    "purchaseStatusUnknown": "Unknown error when trying to purchase the subscription. Consider restarting this app."
+    "purchaseStatusUnknown": "Unknown error when trying to purchase the subscription. Consider restarting this app.",
+
+    "landscapeHeading": "You can manage your Ubuntu instances remotely with {landscapeLink}.",
+    "@landscapeHeading": {
+        "placeholders": {
+            "landscapeLink": {
+                "type": "String"
+            }
+        }
+    },
+    "landscapeQuickSetup": "Quick Setup (Recommended)",
+    "landscapeQuickSetupHint": "Provide your Landscape enrollment information",
+    "landscapeFQDNLabel": "Landscape FQDN",
+    "landscapeFQDNError": "Invalid URI",
```

Comment:
We can be a bit more descriptive here, like `Invalid URI. Format should be HOST.DOMAIN or IP_ADDR.`